r-package/grf/tests/testthat/test_print.R

```diff
@@ -31,3 +31,15 @@ test_that("printing a forest with one regressor is successful", {
   capture_output(print(q.tree))
   expect_true(TRUE)
  })
+
+ test_that("tuning output printing is successful", {
+  p = 4
```

Comment:
Maybe we could make this example even smaller? I've been thinking about how to prevent the R test time from getting too long.